subprojects/distributions/src/changes/accepted-public-api-changes.json

```diff
@@ -1,4 +1,9 @@
 {
     "acceptedApiChanges": [
+        {
+            "type": "org.gradle.api.publish.ivy.plugins.IvyPublishPlugin",
+            "member": "Constructor org.gradle.api.publish.ivy.plugins.IvyPublishPlugin(org.gradle.internal.reflect.Instantiator,org.gradle.api.internal.artifacts.configurations.DependencyMetaDataProvider,org.gradle.api.internal.file.FileResolver,org.gradle.api.publish.internal.ProjectDependencyPublicationResolver,org.gradle.api.internal.file.FileCollectionFactory,org.gradle.api.internal.attributes.ImmutableAttributesFactory)",
+            "acceptation": "Change constructor of incubating plugin"
+        }
     ]
```

Comment:
:x: This should not be necessary. `@Inject` constructors should not be treated as public API (otherwise the fix would be to add the `@since` annotation here).

It's a bug in the `@since` check rule. Can you fix this here:
https://github.com/gradle/gradle/blob/4bf7509a9e746d580564a6d510ceefc5eebb8951/buildSrc/src/main/groovy/org/gradle/binarycompatibility/rules/SinceAnnotationMissingRule.java#L130

Should be: `if (isDeprecated(constructor) || isInject(constructor)) {`